tasks/RedHat.yml

```diff
@@ -1,11 +1,22 @@
 ---
 # tasks file for upgrade (RedHat)
 
-- name: Upgrade all packages (RedHat)
+- name: Upgrade all packages (RedHat) (dnf)
+  ansible.builtin.dnf:
+    name: '*'
```

Comment:
Is there a reason we are using single quotes here (`'*'` vs. `"*"`) and below?

Reply:
I basically just copied the `ansible.builtin.yum` task and changed it as the arguments we need are the same. I'm not sure if there was a specific rationale for using single quotes originally though. I assume it was because the example in the [module documentation](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/yum_module.html#examples) uses single quotes.

Reply:
That's pretty much what I figured.  Where I was going was should our preference for double-quotes over single-quotes be applied here and anywhere else where it doesn't really make a difference syntactically?

Reply:
It doesn't make a difference syntactically (in this case). I can update these but I'd like us to be consistent org-wide if we go that route. Double quotes allow escaped characters to be used so that kind of change should be verified before blanket changing in a given project.

Reply:
I don't have a preference, except that I agree we should try to be consistent.